### DISCUSSION TITLE
Special name thing in terminal

### DISCUSSION THREAD

**Discussion by mcfora** [prasanthrangan/hyprdots#2088] [Q&A] [answered]
How do i get this 
![241231_18h48m37s_screenshot](https://github.com/user-attachments/assets/ca4c7859-1221-493c-9c07-44f2ad2c9ffd)
 pls it looks rlly cool my hyprdots didnt include this  (im using fish shell btw)

**Comment by kRHYME7** [comment, marked as answer]
Try this

https://github.com/starship/starship

https://github.com/IlanCosman/tide

hyprdots is originally zsh. Some contributors merged the fish shell with this prompt `>` . 

By the way that "thing" is called a shell prompt.

I would recommend starship as I think it is already installed in your system when you choose fish.

**Reply by mcfora** [reply]
ok will do thx
